test/gtest/ucp/test_ucp_am.cc

```diff
@@ -296,6 +296,13 @@ UCS_TEST_P(test_ucp_am, send_process_am_release)
     do_send_process_data_test(UCP_RELEASE, 0, 0);
 }
 
+UCS_TEST_P(test_ucp_am, send_process_iov_am_64k_size)
+{
+    ucs::detail::message_stream ms("INFO");
```

Comment:
why need to create "ms" ?

Reply:
I copied the code from `send_process_iov_am`. I thought it's a required one before call `do_send_process_data_iov_test`. I will remove it.